src/main/webapp/app/overview/course-score-calculation.service.ts

```diff
@@ -54,7 +54,7 @@ export class CourseScoreCalculationService {
         return scores;
     }
 
-    private round(value: any, exp: number) {
+    private static round(value: any, exp: number) {
```

Comment:
Why does this method have to be static? Especially `private static` seems strange to me.

Reply:
This was suggested by IntelliJ, I don't mind if it is static or not. I guess they propose it to make explicit that no state of the object is changed and that it is a pure function

Reply:
Ah ok, I understand